src/api/Data/ContainerDetail.cs

```diff
@@ -1,4 +1,6 @@
-using System;
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+

```

Comment:
Should we consistent between blank lines after the statement? I favor having a blank line.